### PR TITLE
[Tyra2] Reorganize the file dates

### PR DESCRIPTION
**Pull request by Wolf3s**

### CODE REVIEW

**Comment by h4570** [comment]
That is a surprise!
Thank you for your contribution!

Please check out the ROADMAP.TXT file from `tyrav2-develop` branch
We discussed about putting `2020-2022` everywhere, but it can be `2022` everywhere also.

**Comment by Wolf3s** [comment]
> That is a surprise! Thank you for your contribution!
> 
> Please check out the ROADMAP.TXT file from `tyrav2-develop` branch We discussed about putting `2020-2022` everywhere, but it can be `2022` everywhere also.

It will maintained 2020-2022 however newer files should be only 2022

File: engine/inc/audio/audio.hpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # Copyright 2020-2022, Tyra - https://github.com/h4570/tyrav2
 # Licenced under Apache License 2.0
 # Sandro Sobczyński <sandro.sobczynski@gmail.com>
-# Wellinator Carvalho <wellcoj@gmail.com>
+# Wellinaton Carvalho <wellcoj@gmail.com>
```
Comment (h4570) [inline engine/inc/audio/audio.hpp:9]: Typo, please fix all of `Wellinaton` typos via CTRL+F

File: engine/inc/loaders/texture/png_loader.hpp [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #   |     \/   ____| |___|
 #   |     |   |   \  |   |
 #-----------------------------------------------------------------------
-# Copyright 2020, tyra - https://github.com/h4570/tyra
+# Copyright 2022, tyra - https://github.com/h4570/tyra
```
Comment (h4570) [inline engine/inc/loaders/texture/png_loader.hpp:6]: Hmm I dont get it.
We should be consistent with this
`2022` or `2020-2022` (the same for other places)

File: engine/inc/loaders/3d/builder/mesh_builder_material_data.hpp [hunk 3]
```diff
@@ -4,7 +4,7 @@
 #   |     \/   ____| |___|
 #   |     |   |   \  |   |
 #-----------------------------------------------------------------------
-# Copyright 2020, tyra - https://github.com/h4570/tyra
+# Copyright 22020 - 2022022, tyra - https://github.com/h4570/tyra
```
Comment (h4570) [inline engine/inc/loaders/3d/builder/mesh_builder_material_data.hpp]: Typo

File: ROADMAP.txt [hunk 4]
```diff
@@ -12,6 +12,9 @@
 --- Wellinator
 - [General] Physics and stuff from threeJS
 
+--- Wolf3s
+- [General] All Copyrights to 2020-2022 and add clang format check(Codacy)
```
Comment (h4570) [inline ROADMAP.txt:15]: Cool idea!

File: engine/inc/debug/debug.hpp [hunk 5]
```diff
@@ -3,7 +3,7 @@
 #   |     \/   ____| |___|
 #   |     |   |   \  |   |
 #-----------------------------------------------------------------------
-# Copyright 2022, tyra - https://github.com/h4570/tyra
+# Copyright 2020 - 2022, tyra - https://github.com/h4570/tyra
```
Comment (h4570) [inline engine/inc/debug/debug.hpp:6]: Could you include #117 for these changes?

**Review by h4570 — CHANGES_REQUESTED** [review]
Small mistakes

**Comment by h4570** [comment]
> > That is a surprise! Thank you for your contribution!
> > Please check out the ROADMAP.TXT file from `tyrav2-develop` branch We discussed about putting `2020-2022` everywhere, but it can be `2022` everywhere also.
> 
> It will maintained 2020-2022 however newer files should be only 2022

So how about putting 2022 everywhere? This should be simple and have same "standard" for every file.

File: engine/inc/debug/debug.hpp (hunk 5 above)
Comment (Wolf3s) [reply engine/inc/debug/debug.hpp:6]: Sure

**Comment by h4570** [comment]
Due to inactivity - I fixed it by myself

**Comment by Wolf3s** [comment]
> Due to inactivity - I fixed it by myself

Yeah but i stopped becuase of the merge and patience

**Pull request closed without merging**